src/software/git/github/CI.md

```diff
@@ -0,0 +1,79 @@
+# 持续集成介绍
+
+## 概念
+
+持续集成( Continuous Integration )指的是，频繁地(一天多次)将代码集成到主干，每次集成都通过自动化的构建(包括编译，发布，自动化测试)来验证，从而尽早地发现集成错误。
+
+## 目的
+
+让产品可以快速迭代，同时还能保持高质量。
+就是说每完成一个完整的部分，就向下个环节交付，发现问题可以马上调整。使得问题不会放大到其他部分和后面的环节。
+
+## 核心措施
+
+它的核心措施是，代码集成到主干之前，必须通过自动化测试。只要有一个测试用例失败，就不能集成。
+
+## 优点
+
+1. 提高开发效率
+1.  快速发现并定位 Bugs

```

Comment:
remove extra space.

AGAIN, I don't think this issue will exsit if you DO run the yarn lint command.

Do as I told you!